README.md

```diff
@@ -18,7 +18,7 @@ The [Step by Step Guide](https://github.com/hyrise/hyrise/wiki/Step-by-Step-Guid
 
 ## Native Setup
 You can install the dependencies on your own or use the install.sh script (**recommended**) which installs all of the therein listed dependencies and submodules.
-The install script was tested under macOS High Sierra and Ubuntu 18.04 (apt-get).
+The install script was tested under macOS High Sierra and Ubuntu 19.04 (apt-get).
```

Comment:
We can probably verify that the install script works with a fresh macOS Mojave today.